app/assets/javascripts/discourse/app/components/sidebar.hbs

```diff
@@ -1,12 +1,32 @@
 <DSection
   @pageClass="has-sidebar"
   @id="d-sidebar"
-  @class="sidebar-container"
+  @class={{concat-class "sidebar-container" this.joinedCustomCssClasses}}
   @scrollTop={{false}}
 >
+
+  {{#each this.topSidebarButtons as |button|}}
```

Comment:
Is there a need for us to support multiple buttons and if so, how will the multiple buttons look when rendered?